### DISCUSSION TITLE
Basic translation (/trust)

### DISCUSSION THREAD

**Discussion by KnightWorldfr** [GriefPrevention/GriefPrevention#2374] [Ideas]
if I run the /trust command (and containertrust accesstrust permissiontrust), a message appears : 
(/trust (player) Grants a player permission to build. See also /untrust,...). 
Can I modify this message to translate it into french? I can't find it in the message.yml file. TY :)

**Comment by Jikoo** [comment]
This is unfortunately not translatable now, at least not via GP's messages system. It is the usage message for invalid command execution declared in the plugin.yml.
https://github.com/GriefPrevention/GriefPrevention/blob/1e5f48ab446f1de03c8b5477dbf3f08923ca0996/src/main/resources/plugin.yml#L23
To edit it, you will have to open the jar (for some tools like Windows file explorer, rename from .jar to .zip), edit the plugin.yml to your liking, and replace the file in the archive.

**Reply by KnightWorldfr** [reply]
I was advised on the Discord server to open a GitHub ticket so you can try to separate these messages so they can be translated in the future. 

![message](https://github.com/user-attachments/assets/cd2a958d-8222-40ae-bf58-9c393e21c9eb)

**Reply by Jikoo** [reply]
That would be helpful. This isn't an issue though, it's a discussion. We may remember it exists, but discussions have a lot less discoverability in terms of "things that need resolution."
